polus-vector-label-plugin/src/main.py

```diff
@@ -0,0 +1,101 @@
+from bfio import  BioWriter , OmeXml,JARS,LOG4J
+#from bfio import  BioWriter
+import bioformats
+import javabridge as jutil
```

Comment:
These can be removed. In the `2.0.0a4-slim-buster` container, there is no bioformats nor javabridge.